packages/x-components/src/x-modules/search-box/components/search-button.vue

```diff
@@ -145,24 +145,34 @@ search term and click the Search button, the search term is displayed as a messa
 
 _Type any term in the input field and then click the Search button to try it out!_
 
-```vue
+```vue live
 <template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work